app/experimenter/experiments/constants/nimbus.py

```diff
@@ -409,6 +413,32 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN = NimbusTargetingConfig(
+    name="Infrequent user (need default, has pin)",
+    slug="infrequent_user_need_default_has_pin",
+    description="Between 1 & 6 days activity in past 28 days need default w/ pin",
+    targeting=("{infrequent_user} && {need_default} && {has_pin} || {sticky}").format(
+        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
+        core_need_default=TARGETING_NEED_DEFAULT,
```

Comment:
s/core_need_default/need_default

Reply:
Is there a better way in python to just use the variable instead of needing to define yet another term that we introduce error?

Reply:
I think we're using python 3.6+ so we can f it?

Reply:
I haven't used it myself, but looks like there's some existing usage, e.g., https://github.com/mozilla/experimenter/blob/c283826781dde5f81bc51c6e564973fde459fd31/app/experimenter/experiments/constants/nimbus.py#L356-L357

Reply:
That should work, though I worry it makes it less readable (though I agree the current implementation is more error prone).

Reply:
https://github.com/mozilla/experimenter/issues/7322